--- librpcsecgss-0.1/configure.in.orig	2004-10-22 19:24:27.000000000 +0200
+++ librpcsecgss-0.1/configure.in	2005-02-28 22:03:09.848455160 +0100
@@ -61,8 +61,8 @@
   fi
   if test "$K5CONFIG" != ""; then
     if test -f $dir/include/gssapi/gssapi_krb5.h -a \
-            \( -f $dir/lib/libgssapi_krb5.a -o \
-	    -f $dir/lib/libgssapi_krb5.so \) ; then
+            \( -f $dir/lib/libgssapi_krb5.so -o \
+	    -f $dir/lib64/libgssapi_krb5.so \) ; then
        AC_DEFINE(HAVE_KRB5)
        KRBDIR="$dir"
        K5VERS=`$K5CONFIG --version | awk '{split($4,v,"."); print v[[1]]v[[2]]v[[3]] }'`
@@ -82,7 +82,7 @@
     dnl of Heimdal Kerberos on SuSe
     elif test \( -f $dir/include/heim_err.h -o\
     		 -f $dir/include/heimdal/heim_err.h \) -a \
-              -f $dir/lib/libroken.a; then
+              \( -f $dir/lib/libroken.so -o -f $dir/lib64/libroken.so \) ; then
        AC_DEFINE(HAVE_HEIMDAL)
        KRBDIR="$dir"
        K5VERS=`$K5CONFIG --version | head -1 | awk '{split($2,v,"."); print v[[1]]v[[2]]v[[3]] }'`
--- librpcsecgss-0.1/src/Makefile.am.orig	2004-10-22 19:24:29.000000000 +0200
+++ librpcsecgss-0.1/src/Makefile.am	2005-02-28 22:15:01.340291960 +0100
@@ -1,4 +1,5 @@
 lib_LTLIBRARIES = librpcsecgss.la
+librpcsecgss_la_LIBADD = $(KRBLIB)
 librpcsecgss_la_SOURCES = \
 	auth_gss.c authgss_prot.c svc.c svc_run.c svc_auth.c \
 	svc_auth_none.c svc_auth_unix.c svc_auth_gss.c \
