--- librpcsecgss-0.7/configure.in.orig	2005-12-05 17:32:46.000000000 +0100
+++ librpcsecgss-0.7/configure.in	2005-12-09 20:36:20.189112720 +0100
@@ -12,10 +12,8 @@
 AC_PROG_RANLIB
 
 # Checks for libraries.
-PKG_CHECK_MODULES(GSSAPI, libgssapi, [],
-  [AC_MSG_ERROR([Unable to locate information required to use libgssapi.
-  If you have pkgconfig installed, you might try setting environment
-  variable PKG_CONFIG_PATH to /usr/local/lib/pkgconfig])])
+AC_CHECK_LIB(gssapi, main, [GSSAPI_LIBS="-lgssapi"], [AC_MSG_ERROR([Unable to locate libgssapi])])
+AC_SUBST(GSSAPI_LIBS)
 
 # Checks for header files.
 AC_HEADER_STDC
